### PR TITLE
AGOV-EN, PHNI-JP cards added

### PR DESCRIPTION
**Pull request by pokeboyexn**

### CODE REVIEW

File: scripts/VJMP-JP/c100200244.lua
Comment (Lyris12) [inline scripts/VJMP-JP/c100200244.lua:1]: Already done: #1191

**Review by Lyris12 — CHANGES_REQUESTED** [review]

File: scripts/VJMP-JP/c100200244.lua
Comment (pokeboyexn) [reply scripts/VJMP-JP/c100200244.lua:1]: Deleted

**Review by Lyris12 — CHANGES_REQUESTED** [review]
AGOV-EN cards 091 and 092 have their scripts swapped

**Comment by pokeboyexn** [comment]
@Lyris12 Thanks for checking. Corrected.

File: scripts/PHNI-JP/c101203029.lua [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		e1:SetCondition(s.rcon)
+		e1:SetReset(RESET_EVENT+RESETS_STANDARD)
+		tc:RegisterEffect(e1)
+		if tc:IsType(TYPE_TRAPMONSTER) then
+			local e2=e1:Clone()
+			e2:SetCode(EFFECT_DISABLE_TRAPMONSTER)
+			tc:RegisterEffect(e2)
+		end
+	end
+end
```
Comment (a597449807) [inline scripts/PHNI-JP/c101203029.lua:68]:
```suggestion
	--cancel target
	local e2=Effect.CreateEffect(c)
	e2:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_CONTINUOUS)
	e2:SetCode(EVENT_CHANGE_POS)
	e2:SetCondition(s.recon)
	e2:SetOperation(s.reop)
	c:RegisterEffect(e2)	
end
```

File: scripts/PHNI-JP/c101203029.lua [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+function s.distg(e,tp,eg,ep,ev,re,r,rp,chk,chkc)
+	if chkc then return chkc:IsOnField() and chkc:IsControler(1-tp) and aux.NegateAnyFilter(chkc) end
+	if chk==0 then return Duel.IsExistingTarget(s.disfilter,tp,0,LOCATION_MZONE,1,nil) end
+	Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_DISABLE)
+	local g=Duel.SelectTarget(tp,s.disfilter,tp,0,LOCATION_MZONE,1,1,nil)
+	Duel.SetOperationInfo(0,CATEGORY_DISABLE,g,1,0,0)
+end
+function s.disop(e,tp,eg,ep,ev,re,r,rp)
+	local c=e:GetHandler()
+	if not c:IsRelateToEffect(e) or c:IsDefensePos() then return end
```
Comment (a597449807) [inline scripts/PHNI-JP/c101203029.lua:50]:
```suggestion
	if not c:IsRelateToEffect(e) or c:IsDefensePos() or not c:IsCanChangePosition() then return end
```

File: scripts/PHNI-JP/c101203029.lua [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		tc:RegisterEffect(e1)
+		if tc:IsType(TYPE_TRAPMONSTER) then
+			local e2=e1:Clone()
+			e2:SetCode(EFFECT_DISABLE_TRAPMONSTER)
+			tc:RegisterEffect(e2)
+		end
+	end
+end
+function s.rcon(e)
+	return e:GetOwner():IsHasCardTarget(e:GetHandler())
```
Comment (a597449807) [inline scripts/PHNI-JP/c101203029.lua:70]:
```suggestion
	return e:GetOwner():IsHasCardTarget(e:GetHandler()) and e:GetOwner():IsDefensePos()
```

File: scripts/PHNI-JP/c101203029.lua [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+end
+
```
Comment (a597449807) [inline scripts/PHNI-JP/c101203029.lua:72]:
```suggestion
	--cancel target
function s.recon(e,tp,eg,ep,ev,re,r,rp)
	local c=e:GetHandler()
	return c:IsPreviousPosition(POS_DEFENSE) and c:IsFaceup() and c:IsAttackPos()
end
function s.reop(e,tp,eg,ep,ev,re,r,rp)
	local c=e:GetHandler()
	local tc=c:GetFirstCardTarget()
	Card.CancelCardTarget(c,tc)
end
```

File: scripts/PHNI-JP/c101203029.lua [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+function s.initial_effect(c)
+	--disable
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetCategory(CATEGORY_POSITION+CATEGORY_DISABLE)
+	e1:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+	e1:SetCode(EVENT_SPSUMMON_SUCCESS)
+	e1:SetProperty(EFFECT_FLAG_DELAY+EFFECT_FLAG_CARD_TARGET)
+	e1:SetCountLimit(1,id)
+	e1:SetLabelObject(e0)
```
Comment (a597449807) [inline scripts/PHNI-JP/c101203029.lua:12]:
```suggestion
```

**Review by a597449807 — COMMENTED** [review]
修复了根据裁定哥布林降下部队无效对方怪兽后，若这张卡变成攻击表示则会和对象怪兽不再建立效果关系，重新变成守备表示后也不会再无效对方怪兽

**Pull request merged**